pkg/generator/templates.go

```diff
@@ -159,7 +160,7 @@ func main() {
 	if err != nil {
 		logrus.Fatalf("failed to get watch namespace: %v", err)
 	}
-	resyncPeriod := 5
+	resyncPeriod := time.Duration(5)
```

Comment:
Shouldn't' this be a default of 5 seconds:
```Go
resyncPeriod := time.Duration(5) * time.Second
```

Otherwise simply casting it to time.Duration means this is 5ns. 
Duration is just an int64 nanosecond count.
https://golang.org/pkg/time/#Duration